### PR TITLE
Posts frontend - Posts working and displaying on event page, by event ID.

### PR DESCRIPTION
**Pull request by Boschio**

### CODE REVIEW

File: gameon-api/models/posts.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ class Posts {
     }
 
     static async listAllPostsByEventId(eventId) {
-
+        console.log("POST MODEL: ", eventId)
```
Comment (Rebecca-CYLi) [inline gameon-api/models/posts.js:64]: remove console logs

File: gameon-api/models/posts.js [hunk 2]
```diff
@@ -88,7 +90,7 @@ class Posts {
         if (!posts) {
             throw new NotFoundError("No posts found.")
         }
-
+        console.log("POSTS MODEL OBJ: ",posts)
```
Comment (Rebecca-CYLi) [inline gameon-api/models/posts.js:93]: remove console logs

**Review by Rebecca-CYLi — CHANGES_REQUESTED** [review]
remove console logs & add meaningful comments to Posts files

**Comment by LifeBringer** [comment]
Also needs to fill panel as best possible.

File: gameon-api/models/posts.js (hunk 2 above)
Comment (Boschio) [reply gameon-api/models/posts.js]: Pushed rquested changes to posts model.

**Review by Rebecca-CYLi — APPROVED** [review]
console logs fixed, G2G

**Pull request merged**